### ISSUE TITLE
run python file on terminal keeps on executing powershell.

### ISSUE THREAD

**Issue by djkim0516** [microsoft/vscode#132706]
Issue Type: <b>Bug</b>

I prefer to use cmd and I changed all the settings, but it seems to keep on executing powershell.

VS Code version: Code 1.60.0 (e7d7e9a9348e6a8cc8c03f877d39cb72e5dfb1ff, 2021-09-01T10:41:52.311Z)
OS version: Windows_NT x64 10.0.19043
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-6600 CPU @ 3.30GHz (4 x 3312)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.96GB (9.06GB free)|
|Process Argv|--crash-reporter-id 744e4c2a-db08-47ea-b5ca-292137213539|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (10)</summary>

Extension|Author (truncated)|Version
---|---|---
TabOut|alb|0.2.1
vscode-pull-request-github|Git|0.30.0
vsc-python-indent|Kev|1.14.2
kite|kit|0.147.0
vscode-language-pack-ko|MS-|1.60.3
python|ms-|2021.9.1191016588
vscode-pylance|ms-|2021.9.1
jupyter|ms-|2021.8.2041215044
jupyter-keymap|ms-|1.0.0
vscodeintellicode|Vis|1.2.14

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
pythonvspyt602:30300191
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyt639:30300192
pythontb:30283811
pythonvspyt551cf:30345471
pythonptprofiler:30281270
vshan820:30294714
vstes263cf:30335440
pythondataviewer:30285071
pythonvsuse255:30340121
vscod805:30301674
pythonvspyt200:30340761
vscextlang:30333561
binariesv615:30325510
pythonvssor306:30344512
bridge0708:30335490
pygetstartedt2:30360495
bridge0723:30353136
pythonf5test824cf:30361778
pythonvspyt187:30362782
pydsgsc2:30361791
vscus224cf:30358040
vssid140:30363603

```

</details>

**Comment by jmanthony3** [comment]
Have same issue. While editing Python script, striking the keyboard shortcut for the command "python.execInTerminal" runs the script by launching a new terminal. However, that new terminal is PowerShell. My default shell is Command Prompt, and I do not want PowerShell.

Version: 1.60.0 (user setup)
Commit: e7d7e9a9348e6a8cc8c03f877d39cb72e5dfb1ff
Date: 2021-09-01T10:41:52.311Z
Electron: 13.1.8
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Windows_NT x64 10.0.19042

**Comment by IllusionMH** [comment]
I believe that it should be fixed by #132868 in upcoming August Recovery release

**Comment by meganrogge** [comment]
pls confirm that this is fixed in insider's or the recovery release
/duplicate of #132150
